.circleci/ci/src/pipelines/tests/resources/pull-requests/pull-requests-custom-branch.yml

```diff
@@ -0,0 +1,769 @@
+# This configuration has been automatically generated by the CircleCI Config SDK.
+# For more information, see https://github.com/CircleCI-Public/circleci-config-sdk-ts
+# SDK Version: 0.0.0-development
+
+version: 2.1
+setup: false
+commands:
+  cmd-restore-maven-job-cache:
+    parameters:
+      jobName:
+        type: string
+        default: ""
+        description: The job name
+    steps:
+      - restore_cache:
+          keys:
+            - gravitee-api-management-v9-<< parameters.jobName >>-{{ .Branch }}-{{ checksum "pom.xml" }}
+            - gravitee-api-management-v9-<< parameters.jobName >>-{{ .Branch }}
+            - gravitee-api-management-v9-<< parameters.jobName >>
+    description: Restore Maven cache for a dedicated job
+  cmd-save-maven-job-cache:
+    parameters:
+      jobName:
+        type: string
+        default: ""
+        description: The job name
+    steps:
+      - run:
+          name: Exclude all APIM artefacts from cache.
+          command: rm -rf ~/.m2/repository/io/gravitee/apim
+      - save_cache:
+          key: gravitee-api-management-v9-<< parameters.jobName >>-{{ .Branch }}-{{ checksum "pom.xml" }}
+          paths:
+            - ~/.m2
+          when: always
+    description: Save Maven cache for a dedicated job
+  cmd-notify-on-failure:
```

Comment:
Is this necessary for a non-master/support branch?

Reply:
I would say yes. But I'm wondering if this really work as expected. I mean, I don't remember we had notifications when a build of PR failed 🤔 .

Reply:
The branch control is inside the command, which makes sense. But maybe this is something we can improve when starting to segment pipelines.